Code/BasicFilters/include/sitkAdditionalProcedures.h

```diff
@@ -100,6 +100,29 @@ SITKBasicFilters_EXPORT Image PatchBasedDenoising (const Image& image1,
 #endif
 
 
+
+
+    /**
+     * \brief Blurs an image by separable convolution with discrete gaussian kernels. This filter performs Gaussian blurring by separable convolution of an image and a discrete Gaussian operator (kernel).
+     *
+     * This function directly calls the execute method of DiscreteGaussianImageFilter
+     * in order to support a procedural API
+     *
+     * \sa itk::simple::DiscreteGaussianImageFilter for the object oriented interface
+     */
+     SITKBasicFilters_EXPORT Image DiscreteGaussian ( const Image& image1, double variance, unsigned int maximumKernelWidth = 32u, double maximumError =  0.01, bool useImageSpacing = true );
+
```

Comment:
I just noticed that for DiscreteGaussian we are using variance vs. sigma in the SmoothingRecursiveGaussian. I am sure this will cause confusion for people who move between the discrete and recursive implementations. Should we add a \sa to each pointing to the other and saying something about sigma vs. variance? Not sure this will help much as people will likely not read it. What do you think?